pkg/helpers/cluster/helpers.go

```diff
@@ -0,0 +1,39 @@
+package cluster
+
+import (
+	"context"
+
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	clusterclientset "open-cluster-management.io/api/client/cluster/clientset/versioned"
+	clusterapiv1beta1 "open-cluster-management.io/api/cluster/v1beta1"
+)
+
+// GetValidManagedClusterSetBindings returns all bindings found in the namespace.
+func GetValidManagedClusterSetBindings(
```

Comment:
I think this helper should be in the pkg/apis/cluster/v1beta1.

It may have a `getter` interface so you would not need it to rely on client code. In this case, user can implement getter using either informer or controller-runtime